package.json

```diff
@@ -41,6 +42,8 @@
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^13.1.8",
     "@types/jest": "^26.0.23",
+    "@types/react": "^17.0.4",
```

Comment:
we have a type mismatch with the version of react we are using.

Reply:
Why doesn't yarn add the correct version of types based off the dependency? :o